sorting/sorting.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program sorts three numbers.")
+	fmt.Println("Enter the numbers:")
+	var a, b, c float64
+	var max, min float64
```

Comment:
Please sort the numbers "inplace" it means that you shouldn't introduce new variables